bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsVoTableCatalog.scala

```diff
@@ -136,11 +136,14 @@ object GemsVoTableCatalog {
 
   // Sets the min/max magnitude limits in the given query arguments
   protected [gems] def optimizeMagnitudeLimits(criterions: List[GemsCatalogSearchCriterion]): List[MagnitudeConstraints] = {
+    val magConstraints = for {
+        criteria <- criterions
+      } yield criteria.criterion.magConstraints
```

Comment:
Right, as we discussed I'll use the convention of using map for simple cases like this where for with 1-statement may be overkill